packages/loader/container-loader/src/container.ts

```diff
@@ -394,18 +394,27 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
 
     /**
      * {@inheritDoc DeltaManager.readonly}
+     * @deprecated - use readOnlyType
      */
     public get readonly() {
         return this._deltaManager.readonly;
     }
 
     /**
      * {@inheritDoc DeltaManager.readonlyPermissions}
+     * @deprecated - use readOnlyType
```

Comment:
in addition to deprecated, can you please add a note into breaking.md?
That said, if clients can't use it now (if they need to wait for bits with ReadOnlyType to populate through the system before they can actually start using them), then maybe it's better to do in 2 releases or so when we know it's actionable message to consumers.